src/containers/UserSettings/i18n/en.json

```diff
@@ -27,7 +27,6 @@
   "settings.language.option-english": "English",
 
   "settings.binaryDataInPlainTextDisplay.title": "Display binary data in plain text",
-  "settings.binaryDataInPlainTextDisplay.description": "Available starting from version 24.1",
```

Comment:
This description now looks redundant, since current ydb versions are 24-1 and higher